test/evp_test.c

```diff
@@ -66,9 +66,9 @@ struct evp_test_method_st {
  * Linked list of named keys.
  */
 typedef struct key_list_st {
+    struct key_list_st *next;
     char *name;
     EVP_PKEY *key;
-    struct key_list_st *next;
```

Comment:
I like the overhead out of the way first.  Shrug.  Put it back to the last element.  Would like @dot-asm to review before I merge, since this is all his doing :)